projects/arc-docs/src/app/docs/guide/components/backend-integration-doc/backend-integration-doc.component.html

```diff
@@ -1 +1,66 @@
-<p>backend-integration-doc works!</p>
+<div class="intro-wrapper">
+  <div class="1st-phase">
+    <nb-card>
+      <nb-card-header fixed>
+        <h2 class="nb-card-header-title">Overview</h2>
+      </nb-card-header>
+      <nb-card-body>
+        <p>
+          This guide provides the necessary steps to integrate a frontend
+          application with a backend authentication service. It includes
+          boilerplate code to ensure a smooth setup and usage of the
+          authentication service without needing a backend developer.
+        </p>
+      </nb-card-body>
+    </nb-card>
+
+    <nb-card>
+      <nb-card-header fixed>
+        <h2 class="nb-card-header-title">Prerequisites</h2>
+      </nb-card-header>
+      <nb-card-body>
+        <ul>
+          <li>Node.js and npm installed</li>
+          <li>Angular CLI installed</li>
+          <li>PostgreSQL installed</li>
```

Comment:
it can be any datasource 
we even support mysql

Reply:
ok